spec/models/user_spec.rb

```diff
@@ -4,12 +4,84 @@
 describe User do
   it { should have_many :reviews }
   it { should have_many :queue_items }
+  it { should have_many(:follower_connections).class_name('Connection').with_foreign_key 'guide_id' }
+  it { should have_many(:following_connections).class_name('Connection').with_foreign_key 'follower_id' }
   it { should validate_presence_of(:password).on :create }
   it { should validate_length_of(:password).is_at_least 6 }
   it { should validate_presence_of :full_name }
   it { should validate_presence_of :email }
 end
 
+describe '#followers' do
```

Comment:
This method belongs to `describe User do` block. Not sure why you keep putting them outside.